scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixResult.java

```diff
@@ -0,0 +1,34 @@
+package scalafix.interfaces;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+public interface ScalafixResult {
```

Comment:
I was not expecting to see side-effect methods on a `Result` object, which seems like the end of the process. I can't really find the right term for it, but I was thinking about something along
```suggestion
public interface ScalafixEvaluation {
```